xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlay.h

```diff
@@ -118,6 +120,15 @@ class CDVDOverlay
    */
   bool IsOverlayContainerFlushable() { return m_overlayContainerFlushable; }
 
+  /*
+   * \brief Specify if the margins are handled by the subtitle codec/parser.
+   */
+  void SetMarginsHandled(bool marginsHandled) { m_marginsHandled = marginsHandled; }
+
```

Comment:
The way both methods are named depicts the notion of state when, in reality, this overrides behaviour. I'd use something like `setForcedMargins` or `blockMarginOverride`, ... etc. I.e., This is something you set on the codec or parser and don't want to be changed later by the renderer